src/sections/Community/Handbook/BookComponent/BookComponent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import BookComponentWrapper from "./BookComponent.style";
+
+const BookComponent = () => {
+  return (
+    <BookComponentWrapper>
+    <a
```

Comment:
Are we able to use a `<Link>` here? Use Link for site internal links like this one where possible.